### DISCUSSION TITLE
Help (Alot of Errors)

### DISCUSSION THREAD

**Discussion by InHumanEgirl** [colemclaren/ttt#162] [Q&A]
_Hello Guys!_

I tried making my own Gmod MOAT TTT server but i have so many Lua Errors my server wont even start.
I tried everything (asked people to help me for money) and i still cant Fix it.

If you guys want to help me (ofc not for free) add me on DC (Canno#7212) 

I really want to fix it and would appreciate  every little bit of help.

_Sorry for the sorry for the grammatical errors!_

**Comment by ghost** [comment]
COOL GAYS THANKS FOR HELPING WITH THE LEAKED MODE THATS CLEARLY STOLEN
